### PR TITLE
Sign up through Bundle Service

### PR DESCRIPTION
**Pull request by kashizui**
1. ~~Create sign up view~~
2. ~~Validate sign up form~~
3. ~~Add new unverified user to database~~
4. ~~Send email to email address for verification~~
5. ~~Create verify view~~
6. ~~Update user in model as verified~~
7. ~~Create resend verification view~~
8. ~~Redirect unverified users to resend verification~~
9. ~~Move all routes to base `/account/.*`~~
10. ~~Test everything~~

Future: add CAPTCHA?
https://www.google.com/recaptcha/intro/index.html

### CODE REVIEW

**Comment by kashizui** [comment]
@percyliang @klopyrev please review

**Comment by kashizui** [comment]
screen shot 2016-02-26 at 8 48 38 pm
screen shot 2016-02-26 at 8 48 55 pm
screen shot 2016-02-26 at 8 49 29 pm

**Comment by kashizui** [comment]
@percyliang: all tests pass!

File: static/common.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    margin-bottom: -1px;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+}
+
+.form-signin input[type="password"] {
+    margin-bottom: 10px;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+}
```
Comment (percyliang) [inline static/common.css]: missing newline

**Comment by percyliang** [comment]
General question: how is the bootstrap styling going to interact with whatever we have on the codalab-worksheets side of things?  It just seems weird that the OAuth backend is saying so much about the view here.  Although the bootstrap does look nice.

Another question: we currently have no minimum length on password - do we want to do that?

Otherwise, LGTM

**Comment by kashizui** [comment]
I agree it is strange. An alternative is to point to a common CSS style
sheet we host on CodaLab.org. However I tried to keep the styling minimal,
it's only a bit of CSS I borrowed from the examples in the Bootstrap
documentation, on top of bootstrap itself.

Yeah we definitely want a minimum length, thanks for catching that!

**Comment by percyliang** [comment]
Hm, let's keep the bootstrap the way it is then.

**Pull request merged**